spec/statsd_spec.rb

```diff
@@ -1,14 +1,14 @@
 require 'helper'
 require 'timecop'
 
-describe Statsd do
-  class Statsd
+describe Datadog::Statsd do
```

Comment:
Do we want to rename this one too? https://github.com/DataDog/dogstatsd-ruby/blob/49dd6b293a2774620a791c55fdd2f13f79186d4a/spec/statsd_spec.rb#L552